### DISCUSSION TITLE
Product Status "Draft" and "Published"

### DISCUSSION THREAD

**Discussion by ChrisSantiago82** [lunarphp/lunar#1420] [Q&A]
Merry Christmas to all the Lunar people :)

I have a question. I have products on the status "Drag". But those products can be found in the search and also can be bought.
I was wondering what the status "Draft" and "Published" do?

**Comment by alecritson** [comment]
The status is a way for you to allow for "draft" products to exist but not be shown on your storefront. You'll need to manually cater for the status if you don't want them to appear, whether that's via the query builder or through filtering on search pages that use something like Algolia.

```php
Product::status('published')->get();
```

**Comment by ChrisSantiago82** [comment]
Thanks for your answer.
I'm using the Starterkit and the search is based on Scout (to try it out I have started with the database driver).
The index is created and looks good. Status is also there.
But when I try to add a condition, it still returns the non published products.

`
Product::search($this->term)->whereIn('status', ['published'])->paginate(50);
`
